handlers/cassandra/cassandra.go

```diff
@@ -116,6 +116,37 @@ func (h *Handler) GetE(cmd common.GetRequest) (<-chan common.GetEResponse, <-cha
 	dataOut := make(chan common.GetEResponse, len(cmd.Keys))
```

Comment:
Why making channels ? Stuff seems to be sequential (no ansynchro) in your code.
Just create an array and use the idx of your range cmd.keys to fillup a specific cell